linchpin/hooks/built_ins/check_ssh/check_ssh.yaml

```diff
@@ -0,0 +1,6 @@
+---
+- hosts: all
+  gather_facts: false
+  tasks:
+    - name: ping task
```

Comment:
How is this checking ssh?

Reply:
@p3ck : Ansible ping module does a ssh connection to the host. https://docs.ansible.com/ansible/latest/modules/ping_module.html